mindsdb/mindsdb
# [2/5] Docs: Add content for Deployment and Connect to MindsDB

# Summary

This PR adds the content for the `Deployment` and `Connect to MindsDB` groups. All pages have the same content as the current docs and all pages have the identical URLs and paths as the current docs.

<img width="1156" alt="mindsdb-2" src="https://user-images.githubusercontent.com/44352119/201408885-4c8cccf4-d13e-4579-9d9e-558f71efb5f4.png">

## What is added [1/5]
- Add all files currently existing under the Deployment and Connect to MindsDB groups from the current docs

## Testing Instructions 👨‍💻

Download the Mintlify CLI using the following command

```bash
npm i mintlify -g
```

Run the following command inside `/docs-v2` to preview changes locally.

```bash
mintlify dev
```

Check to see that everything runs properly and all content matches the expectations mentioned above and all other specifications.
